lk2nd/device/dts/msm8953/msm8953-huawei-milan.dts

```diff
@@ -0,0 +1,53 @@
+// SPDX-License-Identifier: GPL-2.0-only
+// Copyright (c) 2023, bzy-080408 <Bzy080408@outlook.com>
+
+/dts-v1/;
+
+#include <skeleton64.dtsi>
+#include <lk2nd.dtsi>
+
+/ {
+	qcom,msm-id = <QCOM_ID_MSM8953 0>;
+	qcom,board-id = <8000 0>, <8001 0>, <8002 0>, <8003 0>, <8016 0>,
+			<8017 0>, <8018 0>, <8019 0>, <8032 0>, <8033 0>,
+			<8035 0>, <8048 0>, <8049 0>, <8050 0>, <8051 0>,
+			<8066 0>, <8064 0>, <8067 0>, <8080 0>, <8082 0>,
+			<8083 0>, <8098 0>, <8099 0>, <8114 0>, <8130 0>;
+};
+
+&lk2nd {
+	model = "Huawei Maimang 5 / Nova (Plus) / G9 (Plus)";
+	compatible = "huawei,milan", "qcom,msm8953", "lk2nd,device";
+
```

Comment:
Could you please annotate all of them with (close-to-)mainline dts file names from 8953 fork? This way lk2nd will be able to pick it automatically in generic images later.

```suggestion

	lk2nd,dtb-files = "msm8953-huawei-milan";
	
```

If there is no obvious dts, I usually added a fixme and guessed the name like this:

https://github.com/msm8916-mainline/lk2nd/blob/6461706bb92978923474ac670ef38249dbba4e95/lk2nd/device/dts/msm8916/msm8916-asus-z00e.dts#L15